### PR TITLE
Fix stringr library bug

### PR DESCRIPTION
**Pull request by smthfrmn**
Forgot to load the used library

### CODE REVIEW

File: tests/testthat/test_RFunction.R [hunk 1]
```diff
@@ -14,3 +14,17 @@ test_that("gap_unit mins executes", {
   expected_count <- 0
```
Comment (smthfrmn) [inline tests/testthat/test_RFunction.R:14]: Just add some other unit tests

File: RFunction.R [hunk 2]
```diff
@@ -4,6 +4,7 @@ library("lubridate")
 library("lutz")
 library("sf")
 library("terra")
+library("stringr")
```
Comment (smthfrmn) [inline RFunction.R:7]: oops

**Review by andreakoelzsch — APPROVED** [review]
lets try again, thanks for the fix, smith!

**Comment by andreakoelzsch** [comment]
@smthfrmn your stringr addition has solved it. the app is running as expected again, now with your fix. thanks so much!

**Pull request merged**